scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html

```diff
@@ -282,6 +284,17 @@ <h5>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.14.0-beta3/js/bootstrap-select.min.js" integrity="sha512-yrOmjPdp8qH8hgLfWpSFhC/+R9Cj9USL8uJxYIveJZGAiedxyIxwNw4RsLDlcjNlIRR4kkHaDHSmNHAkxFTmgg==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
 <script>
 
+  function removeVariant(variantId) {
+    fetch(`/managed_variant/${variantId}/remove`, {
+        method: "POST",
+        headers: {
+            "X-CSRFToken": "{{ modify_form.csrf_token._value() }}"
+        }
+    }).finally(() => {
+        location.reload();
```

Comment:
But, hm, this will still clear the filters, folding them away, no?

Reply:
Weird, on my local it works.. 🤔

Reply:
Looks ok on chrome, doesnt work at all on safari. Firefox is in between: it cautiously asks if it's ok to resend, then does so and keeps the filter.
<img width="760" height="397" alt="Screenshot 2025-11-04 at 17 43 41" src="https://github.com/user-attachments/assets/8a7e64be-df83-4b81-9e42-a2744b4a222e" />